langkit/textstat.py

```diff
@@ -29,58 +34,59 @@ def automated_readability_index(text: str) -> float:
     return textstat.textstat.automated_readability_index(text)
 
 
-@register_metric_udf(col_type=String)
+@register_metric_udf(col_type=String, schema_name="text_standard_component")
 def dale_chall_readability_score(text: str) -> float:
     return textstat.textstat.dale_chall_readability_score(text)
 
 
-@register_metric_udf(col_type=String)
+@register_metric_udf(col_type=String, schema_name="text_standard_component")
 def linsear_write_formula(text: str) -> float:
     return textstat.textstat.linsear_write_formula(text)
 
 
-@register_metric_udf(col_type=String)
+@register_metric_udf(col_type=String, schema_name="text_standard_component")
 def gunning_fog(text: str) -> float:
     return textstat.textstat.gunning_fog(text)
 
 
 @register_metric_udf(col_type=String)
 def aggregate_reading_level(text: str) -> float:
```

Comment:
we don't need a schema_name here?

Reply:
I can put it back, but gunning_fog is part of the later text_standard overall readability metric, so I thought I'd remove the individual components?

Reply:
Sorry, I didn't understand the changes, I get it now, all good!